src/Searching.js

```diff
@@ -78,8 +84,8 @@ async function combinedSearch(searchTerm) {
 
     // Create two promises, one for the local search, one for the
     // server-side search.
-    const serverSidePromise = serverSideSearch(searchTerm, undefined, false);
-    const localPromise = localSearch(searchTerm, undefined, false);
+    const serverSidePromise = serverSideSearch(searchTerm, undefined);
```

Comment:
You can also remove `undefined` here and on the next line (unless you find it clearer to explicitly pass `undefined`).